backend/logistica_servicio/gestor_entregas/infraestructura/consumidor.py

```
@@ -0,0 +1,143 @@
+        url = f'http://{USUARIOS_HOSTNAME}:{USUARIOS_PORT}/api/v1/seguridad/gestor_usuarios/r/clientes'
+        response = requests.get(url, verify=False, headers={"Authorization": f"Bearer {token}"})
+        response.raise_for_status()
+        clientes = response.json()
+
+        for cliente in clientes["clientes"]:
+            if cliente["id"] == cliente_id:
+                return {
+                    "address": cliente.get("address"),
+                    "geographic_coordinates": cliente.get("geographic_coordinates")
+                }
+        self.logger.error(f"Error al consultar los clientes: cliente_id no existe")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)